### ISSUE TITLE
Module/Resolve Error with React 17

### ISSUE THREAD

**Issue by AugustSaintFreytag** [inokawa/rich-textarea#22]
**Describe the bug**
Trying to use the rich textarea component fails to compile using Node 16 or 17 and React 17.

```
ERROR in ./node_modules/rich-textarea/lib/index.mjs 1:0-56

Module not found: Error: Can't resolve 'react/jsx-runtime' in '<project>/node_modules/rich-textarea/lib'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```
**To Reproduce**
Import and use the component.

**Expected behavior**
The project should compile.

**Platform:**
 - OS: macOS Monterey 12.3.1
 - Package Version: ^0.14.0
 - React Version: ^17.0.2
 - Node Version: 16.13.0, 17.9.0 (both tested)

**Comment by inokawa** [comment]
It looks like the same issue as https://github.com/facebook/react/issues/20235 .
It has some workarounds.

**Comment by inokawa** [comment]
Fixed in `0.14.2`.
Thank you for reporting it!
